baseline_testing/README.md

```diff
@@ -0,0 +1,40 @@
+# Baseline Testing
+
+This folder contains examples and results for performing testing of baseline controllers for different testcases.
+
+## Baseline Controller and Test Case Implementation
+
+- The ``baseline.py`` module file is utilized for executing the baseline control for each test case. It simplifies the process by setting ``u={}``.
+
+- The test script ``run_baselines.py`` facilitates the execution of a sample baseline test, with the test case name provided as a command-line argument. If not explicitly mentioned, the default test case is assumed to be  ``bestest_air``. If the configuration parameter ``run_user_defined_test`` is set to ``false``, the script simulates predefined test case scenarios outlined in ``config.json``. Alternatively, if it is ``true``, the baseline scenario can be customized by specifying parameters such as ``start_time``, ``warmup_period``, and ``length``. Additionally, the user has the option to save the results of KPIs and measurements by setting both configuration parameter ``save_kpi_results`` and ``save_measurements`` to ``true``.
+
+## Run an Example Baseline Test
+The baseline testing could be conducted in either of the two methods:
+
+- Run the test with command: ``$ make run_baseline_<TestCase>``. Run all the baseline testcases with command: ``$ make run_baseline_all``
```

Comment:
Suggest change to "Run tests for a specific test case with the command..."

I also don't see any description or reference to the configuration file, which would be good to include so a user knows it is needed.

Reply:
Changed.

Added "Configuration File Description" in the readme file.